docs/docs/associations.md

```diff
@@ -635,6 +635,104 @@ Series.hasOne(Video);
 Trainer.hasMany(Series);
 ```
 
+## Creating with associations
 
+New elements of an association can be created in one step, provided all elements are new.
+
+### Creating elements of a "BelongsTo" or "HasOne" association
+
+Consider the following models:
+
+```js
+var Product = this.sequelize.define('Product', {
+  title: Sequelize.STRING
+});
+var User = this.sequelize.define('User', {
+  first_name: Sequelize.STRING,
+  last_name: Sequelize.STRING
+});
+
+Product.belongsTo(User);
+// Also works for `hasOne`
+```
+
+A new `Product` and `User` can be created in one step in the following way:
+
+```js
+return Product.create({
+  title: 'Chair',
+  User: {
+    first_name: 'Mick',
+    last_name: 'Broadstone'
+  }
+}, {
+  include: [ User ]
+});
+```
+
+### Creating elements of a "BelongsTo" association with an alias
+
+The previous example can be extended to support an association alias.
+
+```js
+var Creator = Product.belongsTo(User, {as: 'creator'});
+
+return Product.create({
+  title: 'Chair',
+  creator: {
+    first_name: 'Matt',
+    last_name: 'Hansen'
+  }
+}, {
+  include: [ Creator ]
+});
+```
+
+### Creating elements of a "HasMany" or "BelongsToMany" association
+
+Let's introduce the ability to associate a project with many tags. Setting up the models could look like:
+
+```js
+var Tag = this.sequelize.define('Tag', {
+  name: Sequelize.STRING
+});
+
+Product.hasMany(Tag);
+// Also works for `belongsToMany`.
+```
+
+Now we can create a project with multiple tags in the following way:
+
+```js
+Product.create({
+  id: 1,
+  title: 'Chair',
+  Tags: [
+    {id: 1, name: 'Alpha'},
+    {id: 2, name: 'Beta'}
```

Comment:
Should probably remove `id` from these examples since nested creation currently does not work with existing elements.